docker-compose.yaml

```diff
@@ -7,8 +7,8 @@ services:
       - 8080:8080
       - 5001:5001
   vitalam-node:
-    image: ghcr.io/digicatapult/vitalam-node:v2.6.0
-    command: /bin/sh -c "
+    image: ghcr.io/digicatapult/vitalam-node:v2.7.0
+    entrypoint: /bin/sh -c "
```

Comment:
Now `entrypoint` is `vitalam-node` you don't have to wrap this in a `sh` script or overide `entrypoint`. You can just pass as `command` all the flags etc that appear after `vitalam-node`